cni/network/network_test.go

```diff
@@ -1455,6 +1573,70 @@ func TestPluginSwiftV2MultipleAddDelete(t *testing.T) {
 			wantErr:    true,
 			wantErrMsg: "failed to create endpoint: MockEndpointClient Error : AddEndpoints Delegated VM NIC failed",
 		},
+		{
+			name: "SwiftV2 Partial Add fail with Accelnet VM NIC",
+			plugin: &NetPlugin{
+				Plugin: plugin,
+				nm: acnnetwork.NewMockNetworkmanager(acnnetwork.NewMockEndpointClient(func(ep *acnnetwork.EndpointInfo) error {
+					if ep.NICType == cns.NodeNetworkInterfaceAccelnetFrontendNIC {
+						return acnnetwork.NewErrorMockEndpointClient("AddEndpoints Accelnet VM NIC failed") //nolint:wrapcheck // ignore wrapping for test
+					}
+
+					return nil
+				})),
+				ipamInvoker: NewCustomMockIpamInvoker(map[string]acnnetwork.InterfaceInfo{
+					"eth0": {
+						NICType: cns.DelegatedVMNIC,
+					},
+					"eth0-1": {
+						NICType: cns.NodeNetworkInterfaceAccelnetFrontendNIC,
+					},
+				}),
+				report: &telemetry.CNIReport{},
+				tb:     &telemetry.TelemetryBuffer{},
+				netClient: &InterfaceGetterMock{
+					interfaces: []net.Interface{
+						{Name: "eth0"},
+					},
+				},
+			},
+			args:       args,
+			wantNumEps: 0,
+			wantErr:    true,
+			wantErrMsg: "failed to create endpoint: MockEndpointClient Error : AddEndpoints Accelnet VM NIC failed",
+		},
+		{
+			name: "SwiftV2 Partial Add fail with Infra+Accelnet VM NIC",
```

Comment:
testcase for delete success, fail and partial fail?

Reply:
will do it in the future, ADO is open